ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/marketplace-image/cucushift-installer-rehearse-azure-ipi-marketplace-image-workflow.metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-	"path": "cucushift/installer/rehearse/azure/ipi/workers-marketimage/cucushift-installer-rehearse-azure-ipi-workers-marketimage-workflow.yaml",
+	"path": "cucushift/installer/rehearse/azure/ipi/marketplace-image/cucushift-installer-rehearse-azure-ipi-marketplace-image-workflow.yaml",
```

Comment:
how about renaming `marketplace-image` to `marketplace` to keep consistent with gcp and aws platform.

Reply:
sure, updated.